src/main/java/com/faforever/client/player/PrivatePlayerInfoController.java

```diff
@@ -63,7 +63,8 @@ public class PrivatePlayerInfoController extends NodeController<Node> {
   public Label unlockedAchievementsLabel;
   public Separator separator;
 
-  private final ObjectProperty<ChatChannelUser> chatUser = new SimpleObjectProperty<>();
+  @Getter
```

Comment:
This can't use lombok getter as that is the wrong convention. Thee property method is just playerProperty and the field is just player